package.json

```diff
@@ -32,6 +32,7 @@
     "classnames": "^2.2.6",
     "date-fns": "^3.6.0",
     "diff": "^4.0.1",
+    "govuk-frontend": "^5.7.1",
```

Comment:
Is this needed if the css is being inlined?

Reply:
removed